ansible/playbooks/docker-deploy.yml

```diff
@@ -19,6 +19,12 @@
       set_fact:
         use_cyberglove: true
 
+    - name: create a unit_tests folder if using unit_tests
```

Comment:
this should be separate playbook not section in playbooks to run unit tests

Reply:
Fixed in latest commit